internal/powerplatform/services/tenant_settings/resource_tenant_settings.go

```diff
@@ -48,398 +52,417 @@ func (r *TenantSettingsResource) Schema(ctx context.Context, req resource.Schema
 		MarkdownDescription: "Manages Power Platform Tenant Settings. Power Platform Tenant Settings are configuration options that apply to the entire tenant. They control various aspects of Power Platform features and behaviors, such as security, data protection, licensing, and more. These settings apply to all environments within your tenant. See [Tenant Settings Overview](https://learn.microsoft.com/power-platform/admin/tenant-settings) for more details.",
 		Attributes: map[string]schema.Attribute{
 			"id": schema.StringAttribute{
-				Description:         "Id of the read operation",
-				MarkdownDescription: "Id of the read operation",
-				Computed:            true,
+				Description:         "Tenant ID",
+				MarkdownDescription: "Id of the Power Platform Tenant",
+				Computed:            true, Required: false, Optional: false,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
 			},
 			"walk_me_opt_out": schema.BoolAttribute{
 				Description: "Walk Me Opt Out",
-				Optional:    true, Computed: true,
+				Optional:    true,
 				PlanModifiers: []planmodifier.Bool{
-					boolplanmodifier.UseStateForUnknown(),
+					// boolplanmodifier.UseStateForUnknown(),
```

Comment:
can remove remove commented out boolmodifiers that are not used anymore?

Reply:
done